content/pt-br/docs/tasks/configure-pod-container/migrate-from-psp.md

```diff
@@ -0,0 +1,389 @@
+---
+title: Migrando de PodSecurityPolicy Para Controlador de Admissão Integrado
+reviewers:
+- tallclair
+- liggitt
+content_type: task
+min-kubernetes-server-version: v1.22
+---
+
+<!-- overview -->
+
+Esta página descreve o processo de migração de `PodSecurityPolicies` para 
+o controlador de admissão `PodSecurity` integrado. Isso pode ser feito de maneira 
+eficaz usando uma combinação de modos `dry-run` (ensaio/simulação), `audit` e
+`warn`, embora isso se torne mais difícil se PSPs mutáveis forem usados.
+
+## {{% heading "prerequisites" %}}
+
+{{% version-check %}}
+
+- Garantir que a funcionalidade `PodSecurity` do [portal de funcionalidades]
+(/docs/reference/command-line-tools-reference/feature-gates/#feature-gates-for-alpha-or-beta-features) 
```

Comment:
This link "portal de funcionalidades" is broken. Remove the newline after the brackspaces.